generators/client/templates/angular/src/test/javascript/spec/app/account/settings/_settings.component.spec.ts

```diff
@@ -0,0 +1,125 @@
+import {ComponentFixture, TestBed, async, inject} from '@angular/core/testing';
+import {MockBackend} from '@angular/http/testing';
+import {Http, BaseRequestOptions} from '@angular/http';
+import {Observable} from 'rxjs/Rx';
+import {<%=jhiPrefixCapitalized%>LanguageService} from 'ng-jhipster';
+import {Principal, AccountService, <%=jhiPrefixCapitalized%>LanguageHelper} from '../../../../../../main/webapp/app/shared';
```

Comment:
I think `<%=jhiPrefixCapitalized%>LanguageHelper` is only need when enableTranslation